pyrs/dataobjects/fields.py

```diff
@@ -724,11 +726,12 @@ def coordinates(self) -> np.ndarray:
         return self._field.coordinates  # type: ignore
 
     @staticmethod  # noqa: C901
-    def __to_wksp_and_peaks(filename: str,
-                            peak_tag: str,
-                            projectfile: Optional[HidraProjectFile],
-                            hidraworkspace: Optional[HidraWorkspace],
-                            peak_collection: Optional[PeakCollection]) -> Tuple[HidraWorkspace, PeakCollection]:
+    def __to_pointlist_and_peaks(filename: str,
+                                 peak_tag: str,
+                                 projectfile: Optional[HidraProjectFile],
```

Comment:
if `projectfile` is optional, it should have a explicit default value of `None`. Same goes for the remaining arguments.

Reply:
It is an internal private method. Default values are always handled by the caller and do not need to be repeated here.

Reply:
I'm a bit more paranoid and don't trust the codebase, since ultimately it's created by failing ten-fingered creatures.